DUI3-DX/Sdk/Speckle.Converters.Common.DependencyInjection/ToHost/ToHostConverterWithFallback.cs

```diff
@@ -44,7 +45,7 @@ public object Convert(Base target)
     // Direct conversion if a converter is found
     if (_baseConverter.TryConvert(target, out object? result))
     {
-      return result!;
+      return result.NotNull();
```

Comment:
We can replace this with the `[NotNullWhen(true)]` attribute on `TryConvert`
Then no null check will be required.